src/components/product.vue

```diff
@@ -0,0 +1,163 @@
+<template>
```

Comment:
move products list to `products` component
move remaining html and products component to products page

Reply:
use props check issue #4